playbook/app/pb_kits/playbook/pb_textarea/docs/_textarea_swift.md

```diff
@@ -0,0 +1,45 @@
+<img width="300" alt="Screenshot 2023-06-14 at 1 50 02 PM" src="https://github.com/powerhome/playbook/assets/73671109/aeb5750e-6d20-4c3a-8c3c-700c3fe559b9" style="border: 6px solid  black; border-radius: 40px;">
```

Comment:
```suggestion
![](https://github.com/powerhome/playbook/assets/73671109/aeb5750e-6d20-4c3a-8c3c-700c3fe559b9)
```

Lets keep this standard markdown. We could make it fancier with CSS on the playbook website side. Lets avoid inline styles or custom HTML.

This goes for all the `<img>` tags